src/Compilers/Core/Portable/DiagnosticAnalyzer/DefaultAnalyzerAssemblyLoader.Core.cs

```diff
@@ -117,7 +120,7 @@ public DirectoryLoadContext(string directory, DefaultAnalyzerAssemblyLoader load
                 {
                     // Delegate to the compiler's load context to load the compiler or anything
                     // referenced by the compiler
-                    return null;
+                    return s_compilerLoadContext.LoadFromAssemblyName(assemblyName);
```

Comment:
It might be best to also change the comment on line 131 to refer to "default ALC" instead of "parent ALC".

Reply:
Fixed